### PR TITLE
fix: cleanup unused components

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Ran a evaluation on which components and functions where no longer user.

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

File: packages/shared/src/graphql/squads.ts [hunk 1]
```diff
@@ -287,18 +287,6 @@ export interface SquadEdgesData {
...
-  sourceId: string,
-  memberId: string,
-): Promise<SourceMember> => {
-  const res = await request(graphqlUrl, CHECK_USER_MEMBERSHIP, {
-    sourceId,
-    memberId,
-  });
-
-  return res.member;
-};
```
Comment (capJavert) [inline packages/shared/src/graphql/squads.ts:300]: Seen this one is still there on the API, can we remove it there too or leave it in webapp for future? It's just an export so it's not adding to the bundle size..

File: packages/shared/src/graphql/posts.ts [hunk 2]
```diff
@@ -446,12 +417,6 @@ export const POST_BY_URL_QUERY = gql`
   ${SHARED_POST_INFO_FRAGMENT}
 `;
 
-export const getPostByUrl = async (url: string): Promise<Post> => {
```
Comment (capJavert) [inline packages/shared/src/graphql/posts.ts:449]: Also still available in API as my previous comment.

File: packages/shared/src/graphql/posts.ts [hunk 3]
```diff
@@ -362,16 +343,6 @@ export const dismissPostFeedback = (id: string): Promise<EmptyResponse> => {
...
-export const votePost = (
-  id: string,
-  vote: UserPostVote,
-): Promise<EmptyResponse> => {
-  return request(graphqlUrl, VOTE_POST_MUTATION, {
-    id,
-    vote,
-  });
-};
-
```
Comment (capJavert) [inline packages/shared/src/graphql/posts.ts:374]: Hmm this is not used but should we use it in `useVotePost` since it's the same thing?

**Review by sshanzel — APPROVED** [review]
Always love to see code getting deleted.

This made me think, we can probably organize the files as well since we have now introduced proper folders to contain relevant files.

**Review by capJavert — APPROVED** [review]
Slim it!

File: packages/shared/src/graphql/posts.ts (hunk 3 above)
Comment (rebelchris) [reply packages/shared/src/graphql/posts.ts:374]: Based on requestMethod :)

**Pull request merged**